cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -387,9 +390,12 @@ def clone_docket(
 
             if add_docket_entries:
                 clone_docket_entries(session, docket.pk)
+            print("before audio file")
```

Comment:
Do we need this print statement, or did you add it for debugging purposes and accidentally include it in the commit?

I noticed the other print statements you added to notify users about adding audio files and also one to display the docket data. Those seem helpful.

Reply:
Those were some debugging statements, sorry, will remove them

Reply:
Thanks @grossir